### PR TITLE
Feat code of conduct

### PR DESCRIPTION
**Pull request by JimmyLoloy98**
Join new branch "code of conduct"

### CODE REVIEW

File: public/assets/css/styles.css [hunk 1]
```diff
@@ -43,4 +43,13 @@ main {
 h1,
 h2 {
   @apply font-big;
+}
+
+p {
+  @apply font-small; 
+  @apply text-2xl;
```
Comment (paulotijero) [inline public/assets/css/styles.css:50]: Isn't necessary duplicate `@apply`
```suggestion
  @apply font-small text-2xl; 
```

Comment (JimmyLoloy98) [reply public/assets/css/styles.css:50]: :heavy_check_mark:

File: public/assets/css/styles.css [hunk 2]
```diff
@@ -43,4 +43,12 @@ main {
...
-}
\ No newline at end of file
+}
+
+p {
+  @apply font-small text-2xl;
+}
+
+section {
+  @apply my-16;
```
Comment (paulotijero) [inline public/assets/css/styles.css:53]: Use `my-12` and erase this class in section of `index`

Comment (JimmyLoloy98) [reply public/assets/css/styles.css:53]: :heavy_check_mark:

**Pull request merged**